.cspell.json

```diff
@@ -205,10 +213,12 @@
     "adduser",
     "adehome",
     "aderc",
+    "adlink",
     "adoc",
     "AHCI",
     "AISAN",
     "aliceblue",
+    "alid",
```

Comment:
@kenji-miyake Sorry for the inconvenience. I modified the description.

Reply:
Thank you, but please add the reference links as well.

Reply:
@kenji-miyake I added details in the description. Please refer to them.

Reply:
@d-hashimoto Ah, you used Markdown links. I'm sorry I didn't notice that.
But could you fix them so that people can understand plain text? With only Markdown links, reviewers need to click all the links and find the words.

Reply:
@kenji-miyake Thank you for letting me know. I added plain text to all words in the description. Please refer to them.